frontend: bitbox02 refactoring views

thisconnect opened the pull request.

thisconnect commented:
rebased

thisconnect commented:
rebased

Beerosagos reviewed (approved):
tACK! Very nice, thanks @thisconnect :pray: 

I think the first commit has been already merged into master with a different hash, I think you can remove that. If you have time to polish a bit the commit messages would be nice to have them all separated imho

thisconnect commented on frontends/web/src/routes/device/bitbox02/setup/pairing.tsx:
```diff
@@ -25,32 +27,38 @@ type Props = {
+  useEffect(onChannelHashChanged, [deviceID, onChannelHashChanged]);
+
+  useEffect(() => { // TODO: check useSubscribe instead
+    return channelHashChanged(deviceID, onChannelHashChanged);
```
added some comments

Beerosagos replied:
Great, thanks

thisconnect commented:
>  If you have time to polish a bit the commit messages would be nice to have them all separated imho

updated most commit messages and added a comment in the place where it was a bit unclear.

PTAL

Beerosagos commented:
> > If you have time to polish a bit the commit messages would be nice to have them all separated imho
> 
> updated most commit messages and added a comment in the place where it was a bit unclear.
> 
> PTAL

Looks good! Thanks
